src/main/java/Duke.java

```diff
@@ -1,10 +1,81 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final int LENGTH_OF_BY = 4;
+    private static final int LENGTH_OF_AT = 4;
+    private static final int LENGTH_OF_EVENT = 6;
+    private static final int LENGTH_OF_DEADLINE = 9;
+    private static final int LENGTH_OF_TODO = 5;
+
     public static void main(String[] args) {
+        String raw_input;
```

Comment:
The naming of the attribute raw_input could be made camelCase for standardization.